### PR TITLE
Created a program loop for invalid path creation

### PR DESCRIPTION
**Pull request by maross3**
The program will clear the screen on init, and the while loop keeps user bound to program runtime, should they want to retry. This ensures a lot less up arrows in the users' future. The error handling is good practice for development, but seemed a bit excessive.

### CODE REVIEW

File: peach.py [hunk 1]
```diff
@@ -24,8 +24,8 @@ def throw_err(txt=f"{get_emoticon()} A wild error occured! {get_emoticon()}", cl
 
 # checks if we have enough or too many arguments
 def check_args():
-    if (len(args) < expected_args) or (len(args) > expected_args):
```
Comment (maross3) [inline peach.py:27]: This could be replaced with `!=`

File: peach.py [hunk 2]
```diff
@@ -62,36 +62,57 @@ def create_environment(name, location):
...
     # and lowers the names
     for lang in data['languages']:
         languages.append(lang['name'].lower())
-
+    
+    createdEnvironment = True
     # if we have the right amount of arguments
-    if check_args() is True:
+    
+    while True:
```
Comment (maross3) [inline peach.py:91]: While loop allows user to choose whether they want to re-enter their input

File: peach.py [hunk 3]
```diff
@@ -62,36 +62,57 @@ def create_environment(name, location):
...
+        if check_args() is True:
+            os.system('clear')
```
Comment (maross3) [inline peach.py:93]: Clear the screen for UI pwnage

File: peach.py [hunk 4]
```diff
@@ -62,36 +62,57 @@ def create_environment(name, location):
...
+        return True
     else:
-        # if the path does NOT exist, throw an error
-        throw_err(location + " is not a valid path!")
+        print(location + " is not a valid path!")
+        return False
 
 args = sys.argv # just so I don't have to type 'sys.argv' all the time, hehe
+
+def try_again_invalid_path():
```
Comment (maross3) [inline peach.py:72]: Allow user to try again, defined in this method

File: peach.py [hunk 5]
```diff
@@ -62,36 +62,57 @@ def create_environment(name, location):
...
                 # asks user for folder name and location (to be used in create_environment function)
-                folder_name = str(input('Project name: '))
-                folder_loc = str(input("Where do you want to place your project? (full path): "))
-                create_environment(folder_name, folder_loc)
-            else:
-                # throws an error if we haven't chosen a valid language
-                throw_err("that language doesn't exist!")
+                    folder_name = str(input('Project name: '))
+                    folder_loc = str(input("Where do you want to place your project? (full path): "))
+                    createdEnvironment = create_environment(folder_name, folder_loc)
```
Comment (maross3) [inline peach.py:105]: Condition for breaking loop if the environment was created

**Comment by joolbyte** [comment]
This is awesome, but I am currently working on something really similar to this, hehe. I'm gonna set up a development branch and put it all in there. I feel bad for closing such a detailed and awesome pull request, but I really appreciate your offer to help! You pointed me in the right direction for sure 👍🏻

**Pull request closed without merging**